### PR TITLE
Push to a git branch after develop is published.

### PR DESCRIPTION
**Pull request by kshakir**

### CODE REVIEW

**Comment by kshakir** [comment]
Example build & result:
- https://travis-ci.org/broadinstitute/cromwell/builds/348847333
- https://github.com/broadinstitute/cromwell/tree/ks_publish_complete_publish_complete

Will delete result branch after merging.

**Comment by geoffjentry** [comment]
:+1:

@mcovarr it’s for Jenkins

**Comment by kshakir** [comment]
Comment added

File: src/bin/travis/afterSuccess.sh [hunk 1]
```diff
@@ -7,15 +7,66 @@ echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
...
-  docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
-  sbt $@ +publish dockerBuildAndPush
+    # ensure we are not logging while accessing docker
+    set -x
+
+    docker login -u="$DOCKER_USERNAME" -p="$DOCKER_PASSWORD"
+    sbt $@ +publish dockerBuildAndPush
+}
+
+# Some CI environments want to know when new docker images are published. The do not currently poll dockerhub but do
```
Comment (mcovarr) [inline src/bin/travis/afterSuccess.sh]: They do not

File: src/bin/travis/afterSuccess.sh [hunk 2]
```diff
@@ -7,15 +7,66 @@ echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
...
+# poll github. To help those environments, signal that a new set of docker images have been published to dockerhub by
```
Comment (mcovarr) [inline src/bin/travis/afterSuccess.sh]: has been published

File: src/bin/travis/afterSuccess.sh [hunk 3]
```diff
@@ -7,15 +7,66 @@ echo "TRAVIS_BRANCH='$TRAVIS_BRANCH'"
...
+# updating a well known branch github.
```
Comment (mcovarr) [inline src/bin/travis/afterSuccess.sh]: branch in github

**Comment by kshakir** [comment]
> thanks and sorry to seagull your comment

Terrible english always deserves a seagull. 😉

**Comment by geoffjentry** [comment]
@mcovarr 
seagull-cartoon-animals

**Pull request merged**